visualization.shadertoy/resources/language/resource.language.en_gb/strings.po

```diff
@@ -123,3 +123,39 @@ msgstr ""
 msgctxt "#30029"
 msgid "Channel 3 as *.png"
 msgstr ""
+
+msgctxt "#30030"
+msgid "Shadertoy Visualization"
+msgstr ""
+
+msgctxt "#30031"
+msgid "Incorrect reading of the given JSON file, check whether it is correct and available."
+msgstr ""
+
+msgctxt "#30032"
+msgid "Path to own preset list"
+msgstr ""
+
+msgctxt "#30033"
+msgid "Within here can be an own path defined to a preset list within JSON format. If you leave empty, addon use his own preset file."
```

Comment:
Set the path to your custom preset file. If no file path is configured the add-on uses the default preset file.